packages/react-instantsearch/src/connectors/connectCurrentRefinements.js

```diff
@@ -1,17 +1,22 @@
 import createConnector from '../core/createConnector';
-
+import {PropTypes} from 'react';
 /**
  * connectCurrentRefinements connector provides the logic to build a widget that will
  * give the user the ability to remove all or some of the filters that were
  * set.
  * @name connectCurrentRefinements
  * @kind connector
+ * @propType {function} transformItems - If provided, this function can be used to modified the `items` provided prop of the wrapped component (ex: for filtering or sorting items). this function take the `items` prop as a parameter and expect it back in return.
```

Comment:
I ask this because if I add it, there's nothing displayed on the doc. I was wondering how we can add the fact that it's optional when there's no default value like here.
See:

![image](https://cloud.githubusercontent.com/assets/6885378/21802763/5378861c-d727-11e6-99f5-39366e672a9d.png)